CHECKENFORCER

```diff
@@ -0,0 +1,3 @@
+format: v0.1-alpha
```

Comment:
Similar question as https://github.com/Azure/azure-sdk-for-python/pull/7331/files#r325768312